docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/template-for-javascript-tracker-v3/tag-template-guide/index.md

```diff
@@ -4,27 +4,27 @@ date: "2021-11-18"
+This template implements the [Snowplow JavaScript tracker v3](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/index.md). The template supports all the features of the tracker, with a few exceptions due to the limitations of custom templates’ sandboxed JavaScript.
 
 ## Install the template
 
 To **install the template**, browse to **Templates** in the Google Tag Manager user interface.
 
-Under **Tag Templates**, click **Search Gallery**, and type `snowplow analytics v3` into the gallery overlay search bar.
+Under **Tag Templates**, click **Search Gallery**, and type `Snowplow v3` into the gallery overlay search bar.
 
-![](images/Screenshot-2021-11-18-at-10.30.35.png)
+![search Snowplow v3 in GTM gallery](images/search_snowplow_v3.png)
 
-Click the **Snowplow Analytics v3** template name, and then click **Add to Workspace** in the next screen. Review the permissions and click **Add** to finalize the import.
+Click the **Snowplow v3** template name, and then click **Add to Workspace** in the next screen. Review the permissions and click **Add** to finalize the import.
 
-After importing the template, you can follow the normal process of creating a **new tag** in Google Tag Manager, and the **Snowplow Analytics v3** template will be listed among the **Custom** tag types you can choose from.
+After importing the template, you can follow the normal process of creating a **new tag** in Google Tag Manager, and the **Snowplow v3** template will be listed among the **Custom** tag types you can choose from.
 
 ## Caveats
 
 To begin with, some of the caveats of using the Custom Template.
 
 - Any methods that require the parsing of HTML elements (e.g. link tracking filter functions, cross domain linking) will not work and are thus disabled.
 - Automatic error tracking does not work due to lack of support for the `ErrorEvent` API.
-- There is no implementation for the [standard ecommerce](https://github.com/snowplow/snowplow/wiki/2-Specific-event-tracking-with-the-Javascript-tracker#ecommerce) events. Users are encouraged to implement the [enhanced ecommerce](https://github.com/snowplow/snowplow/wiki/2-Specific-event-tracking-with-the-Javascript-tracker#enhanced-ecommerce) setup instead.
+- There is no implementation for the [standard ecommerce](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/#ecommerce-tracking) events. Users are encouraged to implement the [enhanced ecommerce](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/#enhanced-ecommerce-tracking) setup instead.
```

Comment:
Use a markdown link please (see https://github.com/snowplow/snowplow.github.io#links)

Reply:
(here and below)

Reply:
You can try running `./make-links-validatable.sh` to do that automatically